src/views/mainLayout.js

```diff
@@ -14,6 +14,27 @@ const mainLayout = data => {
   const metaDescription = data.meta?.socialDescription ?? getMessage('meta-desc-2')
   const pageUrl = `${AppConstants.SERVER_URL}${data.pathname ?? '/'}`
 
+  const shouldShowRecruitmentBanner = () => {
+    // Only show if ENVs are set and user language is set to "en"
+    if (
+      AppConstants.RECRUITMENT_BANNER_TEXT &&
+      AppConstants.RECRUITMENT_BANNER_LINK &&
+      getLocale().includes('en')
+    ) {
+      return true
+    }
+    return false
+  }
+
+  const recruitmentBanner = () => ` 
+    <div role="banner" class="recruitment-banner" hidden>
```

Comment:
a11y note: Added ["banner" type aria role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/banner_role) to the element

Reply:
Removed in ad11567d72b884d0a90d9cd5b9a365d29d1569cb